examples/integration-tutorials/routing-requests-based-on-message-content/test/health_care_service_test.bal

```diff
@@ -63,8 +64,8 @@ function testReservation() {
     var response = clientEP->post("/healthcare/categories/surgery/reserve", req);
     if (response is http:Response) {
         test:assertEquals(response.statusCode, 200, msg = "Reserve-Appointment service did not respond with 200 OK signal!");
-        string|error receivedPayload = response.getTextPayload();
-        if(receivedPayload is string) {
+        string | error receivedPayload = response.getTextPayload();
```

Comment:
@Asitha These spaces between OR operator added by the auto format of VS code. Is it acceptable?

Reply:
According to https://github.com/ballerina-platform/ballerina-lang/tree/master/docs/style_guide this is the accepted way.